src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -388,6 +389,8 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
             var context = new Dictionary<string, object>()
             {
                 { "invocationId", invocationContext.ExecutionContext.InvocationId },
+                { "functionDirectory", invocationContext.ExecutionContext.FunctionDirectory },
```

Comment:
I wonder if we shouldn't add a new 'executionContext' member here that includes all 3 properties. We can leave the invocationId as is for back compat, but going forward it'd be good to have all the properties in a single object I think.